src/config/globals.js

```diff
@@ -20,6 +21,7 @@ module.exports = function () {
   global.GENERIC_HELPERS_PATH = GENERICS_FILES_PATH + "/helpers"
   global._ = require("lodash")
   global.UTILS = require(GENERIC_HELPERS_PATH + "/utils")
```

Comment:
@MallanagoudaB UTILS is already defined why you are defining again? Please remove the latest one and check unnati service how to call the utils function